bin/index.js

```diff
@@ -86,6 +96,12 @@ module.exports = function(Aquifer, AquiferGitConfig) {
       return false;
     }
 
+    // Validate custom signature options.
+    if (!options.name != !options.email) {
+      callback('Both name and email options are required for a custom commit signature.');
+      return false;
```

Comment:
Yeah, it looks like Aquifer doesn't care what extensions return from the run method.